dbschema/default.esdl

```diff
@@ -0,0 +1,24 @@
+module default {
+  type Person {
+    required property name -> str;
+    multi link media -> Media;
+  }
+
+  type Media {
+    required property title -> str;
+  }
```

Comment:
Save this for when we do media and users

Reply:
Da